packages/callout-block/src/CalloutBlock.tsx

```diff
@@ -3,131 +3,62 @@
 import { useBlockAssets, useBlockSettings, useEditorState } from '@frontify/app-bridge';
 import { RichTextEditor } from '@frontify/fondue';
 import '@frontify/fondue-tokens/styles';
-import { Radius, joinClassNames, radiusStyleMap, useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
-import { CSSProperties, FC, createRef, useEffect, useState } from 'react';
+import { joinClassNames, radiusStyleMap, useGuidelineDesignTokens } from '@frontify/guideline-blocks-shared';
```

Comment:
Suggestion: fix import order. 
P.S. This should be the rule of the code of conduct, in my opinion.